core/src/apps/ethereum/keychain.py

```diff
@@ -16,7 +16,7 @@
     from apps.common.keychain import MsgOut, Handler, HandlerWithKeychain
 
     class MsgWithAddressN(MessageType, Protocol):
-        address_n = ...  # type: List[int]
+        address_n: List[int] = ...
```

Comment:
the `= ...` is superfluous when the type annotation is present

Reply:
Addressed in 998eddfeba30f43c6c550f01b717c0c654650ac2